Core/include/Acts/Geometry/VolumeBounds.hpp

```diff
@@ -115,6 +115,12 @@ class VolumeBounds {
       const Transform3* trf = nullptr, const Vector3& envelope = {0, 0, 0},
       const Volume* entity = nullptr) const = 0;
 
+  /// Get the canonical binning values, i.e. the binning values
+  /// for that fully describe the shape's extent
+  ///
+  /// @return vector of canonical binning values
+  virtual std::vector<Acts::BinningValue> canonicalBinning() const = 0;
```

Comment:
Here we could give a default (x,y,z) and document that this is what you get per default, and that it can be overloaded.

Reply:
Yes, it'll make the code a bit cleaner. Will implement now